Tests/Pcap++Test/Tests/TcpReassemblyTests.cpp

```diff
@@ -965,8 +999,16 @@ PTF_TEST_CASE(TestTcpReassemblyIPv6MultConns)
 	PTF_ASSERT_EQUAL(iter->second.connData.srcPort, 35999);
 	PTF_ASSERT_EQUAL(stats.begin()->second.connData.startTime.tv_sec, 1147551795);
 	PTF_ASSERT_EQUAL(stats.begin()->second.connData.startTime.tv_usec, 526632);
-	PTF_ASSERT_EQUAL(stats.begin()->second.connData.endTime.tv_sec, 0);
-	PTF_ASSERT_EQUAL(stats.begin()->second.connData.endTime.tv_usec, 0);
+	PTF_ASSERT_EQUAL(std::chrono::duration_cast<std::chrono::nanoseconds>(
+	                     stats.begin()->second.connData.startTimePrecise.time_since_epoch())
+	                     .count(),
```

Comment:
Can we have a better format here?

Reply:
so just ignore `clang-format`? 🤔

Reply:
I think you can ignore it here as the format seems not perfect.

Reply:
Done in this commit https://github.com/seladb/PcapPlusPlus/pull/1591/commits/b0836404dad63980918c07ad5adeaa918f912977, please let me know if the format looks ok now